examples/tabs/palette-exemplar/tests/containers/__snapshots__/SampleNetworkContainer.test.tsx.snap

```diff
@@ -11,22 +11,12 @@ exports[`SampleNetworkContainer SampleNetworkContainer can render with redux 1`]
     <pre
       class="bp3-code-block"
     >
-      sampleNetwork:{
-  "data": [],
-  "error": null,
-  "loading": false,
-  "success": false
-}
+      simpleNetwork: {}
```

Comment:
Why does it need these props if they can be empty?

Reply:
This is a snapshot that shows what would be displayed for an empty Redux store, which by default has the keys: data, error, loading, success. The update is because of a change in what we want to display for demo purposes.